homeassistant/components/reolink/host.py

```diff
@@ -273,6 +274,18 @@ async def register_webhook(self) -> None:
         webhook_path = webhook.async_generate_path(event_id)
         self._webhook_url = f"{base_url}{webhook_path}"
 
+        if base_url.startswith("https"):
+            ir.async_create_issue(
```

Comment:
What cleans up the issue if it is resolved?

Reply:
The default of `is_persistent` is not mentioned in the dev docs: https://developers.home-assistant.io/docs/core/platform/repairs/
Schould I set `is_persistent=False`?

Reply:
> I thought that if is_persistent was not set a restart of HomeAssistant will clear the issue

Right, but this issue can be resolved by the user without restarting. So, relying on that is incorrect.

Reply:
@frenck I added `ir.async_delete_issue(self._hass, DOMAIN, "https_webhook")` to clean up the issue when a HTTP local adress is used.